docs/installation_guide.md

```diff
@@ -103,16 +103,23 @@ The parameters are described below - note that at the very least, you will need
 
 
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database used for **db_auth**. Change this password for any production use!
```

Comment:
This is not for `db_auth` only, it is for the user who uses local DB, the other auth method like ldap, oidc also need this if they did not use an external database.